scipy/sparse/_base.py

```diff
@@ -557,13 +617,13 @@ def __matmul__(self, other):
         if isscalarlike(other):
             raise ValueError("Scalar operands are not allowed, "
                              "use '*' instead")
-        return self.__mul__(other)
+        return self._mul_dispatch(other)
 
     def __rmatmul__(self, other):
         if isscalarlike(other):
             raise ValueError("Scalar operands are not allowed, "
                              "use '*' instead")
-        return self.__rmul__(other)
+        return self._rmul_dispatcher(other)
```

Comment:
Nitpicking: we have `_mul_dispatch` and `_rmul_dispatcher`. Let's pick one ending and use that for both.

Reply:
Changed.